modules/ROOT/pages/proxy-domain-support.adoc

```diff
@@ -110,18 +111,18 @@ To avoid a conflict when running multiple proxies using the same domain, ensure
 
 === API Gateway Domain Template
 
-If you choose to download your API proxy and configure it manually on your end, you may use the *API Gateway domain template*.
+If you choose to download and manually configure your API proxy on your end, use the *API Gateway domain template*.
 
-The https://anypoint.mulesoft.com/exchange/org.mule.examples/gateway-proxy-domain/[API Gateway domain template] is configured to have a shared HTTP listener configuration ("api-proxy-listener-http") listening on the 8081 port. You can also have a shared HTTPs listener configuration ("api-proxy-listener-https"). 
+The https://anypoint.mulesoft.com/exchange/org.mule.examples/gateway-proxy-domain/[API gateway domain template] is configured to have a shared HTTP listener configuration ("api-proxy-listener-http") listening on the 8081 port. You can also have a shared HTTPs listener configuration ("api-proxy-listener-https"). 
 
-To use either configuraton, you must uncomment code from your API Gateway domain template and configure your TLS context, such as, certificates and passwords. The following list provides the available listeners configurations:
+To use either configuration, you must uncomment code from your API gateway domain template and configure your TLS context, such as, certificates and passwords. The following list provides the available listeners configurations:
```

Comment:
delete "you must"
Unless we tell them something is optional, every instruction is required. Saying only some of them are "must" is inconsistent.

Remove the comma after "such as"